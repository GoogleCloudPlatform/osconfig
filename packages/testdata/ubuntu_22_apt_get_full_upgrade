Inst libpam0g [1.4.0-11ubuntu2.4] (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64])
Conf libpam0g (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64])
Inst libpam-modules-bin [1.4.0-11ubuntu2.4] (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64]) [libpam-modules:amd64 on libpam-modules-bin:amd64] [libpam-modules:amd64 ]
Conf libpam-modules-bin (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64]) [libpam-modules:amd64 ]
Inst libpam-modules [1.4.0-11ubuntu2.4] (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64])
Conf libpam-modules (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [amd64])
Inst libnss-systemd [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) []
Inst libsystemd0 [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) [systemd:amd64 ]
Conf libsystemd0 (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) [systemd:amd64 ]
Inst systemd-sysv [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) [systemd:amd64 ]
Inst libpam-systemd [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) [systemd:amd64 ]
Inst systemd [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Inst udev [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64]) []
Inst libudev1 [249.11-0ubuntu3.12] (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf libudev1 (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Inst libpam-runtime [1.4.0-11ubuntu2.4] (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [all])
Conf libpam-runtime (1.4.0-11ubuntu2.5 Ubuntu:22.04/jammy-updates [all])
Inst libdevmapper1.02.1 [2:1.02.175-2.1ubuntu4] (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst libcryptsetup12 [2:2.4.3-1ubuntu1.2] (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64])
Inst libseccomp2 [2.5.3-2ubuntu2] (2.5.3-2ubuntu3~22.04.1 Ubuntu:22.04/jammy-updates [amd64])
Conf libseccomp2 (2.5.3-2ubuntu3~22.04.1 Ubuntu:22.04/jammy-updates [amd64])
Inst libdevmapper-event1.02.1 [2:1.02.175-2.1ubuntu4] (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst dmsetup [2:1.02.175-2.1ubuntu4] (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst liblvm2cmd2.03 [2.03.11-2.1ubuntu4] (2.03.11-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst dmeventd [2:1.02.175-2.1ubuntu4] (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst lvm2 [2.03.11-2.1ubuntu4] (2.03.11-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Inst libopeniscsiusr [2.1.5-1ubuntu1] (2.1.5-1ubuntu1.1 Ubuntu:22.04/jammy-updates [amd64])
Inst open-iscsi [2.1.5-1ubuntu1] (2.1.5-1ubuntu1.1 Ubuntu:22.04/jammy-updates [amd64])
Inst distro-info-data [0.52ubuntu0.7] (0.52ubuntu0.8 Ubuntu:22.04/jammy-updates [all])
Inst dmidecode [3.3-3ubuntu0.1] (3.3-3ubuntu0.2 Ubuntu:22.04/jammy-updates [amd64])
Inst apt-transport-artifact-registry [1:20240607.00-g1] (1:20241017.00-g1 apt-transport-artifact-registry-stable:apt-transport-artifact-registry-stable [amd64])
Inst cryptsetup-initramfs [2:2.4.3-1ubuntu1.2] (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [all]) []
Inst cryptsetup-bin [2:2.4.3-1ubuntu1.2] (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64]) []
Inst cryptsetup [2:2.4.3-1ubuntu1.2] (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64])
Inst libpackagekit-glib2-18 [1.2.5-2ubuntu2] (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Inst gir1.2-packagekitglib-1.0 [1.2.5-2ubuntu2] (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Inst landscape-common [23.02-0ubuntu1~22.04.3] (23.02-0ubuntu1~22.04.4 Ubuntu:22.04/jammy-updates [amd64])
Inst libldap-2.5-0 [2.5.18+dfsg-0ubuntu0.22.04.2] (2.5.18+dfsg-0ubuntu0.22.04.3 Ubuntu:22.04/jammy-updates [amd64])
Inst libldap-common [2.5.18+dfsg-0ubuntu0.22.04.2] (2.5.18+dfsg-0ubuntu0.22.04.3 Ubuntu:22.04/jammy-updates [all])
Inst libmbim-proxy [1.28.0-1~ubuntu20.04.1] (1.28.0-1~ubuntu20.04.2 Ubuntu:22.04/jammy-updates [amd64]) []
Inst libmbim-glib4 [1.28.0-1~ubuntu20.04.1] (1.28.0-1~ubuntu20.04.2 Ubuntu:22.04/jammy-updates [amd64])
Inst packagekit-tools [1.2.5-2ubuntu2] (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64]) []
Inst packagekit [1.2.5-2ubuntu2] (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Inst pollinate [4.33-3ubuntu2] (4.33-3ubuntu2.1 Ubuntu:22.04/jammy-updates [all])
Inst python3-packaging (21.3-1 Ubuntu:22.04/jammy [all])
Inst snapd [2.65.3+22.04] (2.67.1+22.04 Ubuntu:22.04/jammy-updates [amd64])
Inst sosreport [4.5.6-0ubuntu1~22.04.2] (4.8.2-0ubuntu0~22.04.1 Ubuntu:22.04/jammy-updates [amd64])
Inst xfsprogs [5.13.0-1ubuntu2] (5.13.0-1ubuntu2.1 Ubuntu:22.04/jammy-updates [amd64])
Inst cloud-init [24.3.1-0ubuntu0~22.04.1] (24.4.1-0ubuntu0~22.04.2 Ubuntu:22.04/jammy-updates [all])
Conf libnss-systemd (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf systemd-sysv (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf libpam-systemd (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf systemd (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf udev (249.11-0ubuntu3.15 Ubuntu:22.04/jammy-updates [amd64])
Conf libdevmapper1.02.1 (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf libcryptsetup12 (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64])
Conf libdevmapper-event1.02.1 (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf dmsetup (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf liblvm2cmd2.03 (2.03.11-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf dmeventd (2:1.02.175-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf lvm2 (2.03.11-2.1ubuntu5 Ubuntu:22.04/jammy-updates [amd64])
Conf libopeniscsiusr (2.1.5-1ubuntu1.1 Ubuntu:22.04/jammy-updates [amd64])
Conf open-iscsi (2.1.5-1ubuntu1.1 Ubuntu:22.04/jammy-updates [amd64])
Conf distro-info-data (0.52ubuntu0.8 Ubuntu:22.04/jammy-updates [all])
Conf dmidecode (3.3-3ubuntu0.2 Ubuntu:22.04/jammy-updates [amd64])
Conf apt-transport-artifact-registry (1:20241017.00-g1 apt-transport-artifact-registry-stable:apt-transport-artifact-registry-stable [amd64])
Conf cryptsetup-initramfs (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [all])
Conf cryptsetup-bin (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64])
Conf cryptsetup (2:2.4.3-1ubuntu1.3 Ubuntu:22.04/jammy-updates [amd64])
Conf libpackagekit-glib2-18 (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Conf gir1.2-packagekitglib-1.0 (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Conf landscape-common (23.02-0ubuntu1~22.04.4 Ubuntu:22.04/jammy-updates [amd64])
Conf libldap-2.5-0 (2.5.18+dfsg-0ubuntu0.22.04.3 Ubuntu:22.04/jammy-updates [amd64])
Conf libldap-common (2.5.18+dfsg-0ubuntu0.22.04.3 Ubuntu:22.04/jammy-updates [all])
Conf libmbim-proxy (1.28.0-1~ubuntu20.04.2 Ubuntu:22.04/jammy-updates [amd64])
Conf libmbim-glib4 (1.28.0-1~ubuntu20.04.2 Ubuntu:22.04/jammy-updates [amd64])
Conf packagekit-tools (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Conf packagekit (1.2.5-2ubuntu3 Ubuntu:22.04/jammy-updates [amd64])
Conf pollinate (4.33-3ubuntu2.1 Ubuntu:22.04/jammy-updates [all])
Conf python3-packaging (21.3-1 Ubuntu:22.04/jammy [all])
Conf snapd (2.67.1+22.04 Ubuntu:22.04/jammy-updates [amd64])
Conf sosreport (4.8.2-0ubuntu0~22.04.1 Ubuntu:22.04/jammy-updates [amd64])
Conf xfsprogs (5.13.0-1ubuntu2.1 Ubuntu:22.04/jammy-updates [amd64])
Conf cloud-init (24.4.1-0ubuntu0~22.04.2 Ubuntu:22.04/jammy-updates [all])
