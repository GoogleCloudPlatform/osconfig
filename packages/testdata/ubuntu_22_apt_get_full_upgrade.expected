[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "libpam0g",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.4.0-11ubuntu2.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libpam-modules",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.4.0-11ubuntu2.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libnss-systemd",
        Arch:    "x86_64",
        RawArch: "",
        Version: "249.11-0ubuntu3.15",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd",
        Arch:    "x86_64",
        RawArch: "",
        Version: "249.11-0ubuntu3.15",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "udev",
        Arch:    "x86_64",
        RawArch: "",
        Version: "249.11-0ubuntu3.15",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libudev1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "249.11-0ubuntu3.15",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libpam-runtime",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.0-11ubuntu2.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libdevmapper1.02.1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:1.02.175-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libcryptsetup12",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:2.4.3-1ubuntu1.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libseccomp2",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.5.3-2ubuntu3~22.04.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libdevmapper-event1.02.1",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:1.02.175-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "dmsetup",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:1.02.175-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "liblvm2cmd2.03",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.03.11-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "dmeventd",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:1.02.175-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "lvm2",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.03.11-2.1ubuntu5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libopeniscsiusr",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.1.5-1ubuntu1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "open-iscsi",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.1.5-1ubuntu1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "distro-info-data",
        Arch:    "all",
        RawArch: "",
        Version: "0.52ubuntu0.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "dmidecode",
        Arch:    "x86_64",
        RawArch: "",
        Version: "3.3-3ubuntu0.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "apt-transport-artifact-registry",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:20241017.00-g1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cryptsetup-initramfs",
        Arch:    "all",
        RawArch: "",
        Version: "2:2.4.3-1ubuntu1.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cryptsetup-bin",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:2.4.3-1ubuntu1.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cryptsetup",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2:2.4.3-1ubuntu1.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libpackagekit-glib2-18",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.2.5-2ubuntu3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "gir1.2-packagekitglib-1.0",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.2.5-2ubuntu3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "landscape-common",
        Arch:    "x86_64",
        RawArch: "",
        Version: "23.02-0ubuntu1~22.04.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libldap-2.5-0",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.5.18+dfsg-0ubuntu0.22.04.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libldap-common",
        Arch:    "all",
        RawArch: "",
        Version: "2.5.18+dfsg-0ubuntu0.22.04.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libmbim-proxy",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.28.0-1~ubuntu20.04.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libmbim-glib4",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.28.0-1~ubuntu20.04.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "packagekit-tools",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.2.5-2ubuntu3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "packagekit",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1.2.5-2ubuntu3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pollinate",
        Arch:    "all",
        RawArch: "",
        Version: "4.33-3ubuntu2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "snapd",
        Arch:    "x86_64",
        RawArch: "",
        Version: "2.67.1+22.04",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "sosreport",
        Arch:    "x86_64",
        RawArch: "",
        Version: "4.8.2-0ubuntu0~22.04.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "xfsprogs",
        Arch:    "x86_64",
        RawArch: "",
        Version: "5.13.0-1ubuntu2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cloud-init",
        Arch:    "all",
        RawArch: "",
        Version: "24.4.1-0ubuntu0~22.04.2",
        Source:  packages.Source{},
    },
}